src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
That means that if someone needs translation then they have to pass an entirely new component. I think that the label must be a prop, no matter what it defaults to.